src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -1405,6 +1405,8 @@ trait Definitions extends api.StandardDefinitions {
       case _                  => false
     }
 
+    lazy val InfixAnnotationClass = rootMirror.getRequiredClass("scala.annotation.infix")
```

Comment:
Prefer `getClassIfDefined` to allow the compiler to be able to run with older libraries